tools/tasks/seed/build.html_css.ts

```diff
@@ -4,7 +4,7 @@ import * as merge from 'merge-stream';
 import * as autoprefixer from 'autoprefixer';
 import * as cssnano from 'cssnano';
 import {join} from 'path';
-import {APP_SRC, TMP_DIR, CSS_PROD_BUNDLE, CSS_DEST, APP_DEST, BROWSER_LIST, ENV, getEnvDependencies} from '../../config';
```

Comment:
Maybe a deprecation warning is missing on `getEnvDependencies` method of `SeedConfig` class.